internal/service/backup/find.go

```diff
@@ -136,3 +136,52 @@ func findFrameworkByName(ctx context.Context, conn *backup.Client, name string)
 
 	return output, nil
 }
+
+func FindRestoreTestingPlanByName(ctx context.Context, conn *backup.Client, name string) (*backup.GetRestoreTestingPlanOutput, error) {
```

Comment:
Lowercase/private seems to be the standard for now
```suggestion
func findRestoreTestingPlanByName(ctx context.Context, conn *backup.Client, name string) (*backup.GetRestoreTestingPlanOutput, error) {
```